extensions/Worker.Extensions.Http.AspNetCore/release_notes.md

```diff
@@ -6,8 +6,7 @@
 
 ### Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore <version>
 
-- Updated`Updated Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.Analyzers` 1.0.3
+- Addressing fix for AspNetCoreResponseCookies cookie defaults (#2811)
 
-### Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.Analyzers 1.0.3
+### Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.Analyzers  <version>
```

Comment:
Discussed offline - this change is not yet released so it should still be included in the release notes.

Reply:
So, to confirm, this wasn't released with 1.0.3? https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore.Analyzers/1.0.3